boards/launchxl/Makefile

```diff
@@ -0,0 +1,27 @@
+# Makefile for building the tock kernel for the launchxl development kit
+
+TOCK_ARCH=cortex-m4
+TOCK_BOARD=launchxl
+TARGET=thumbv7em-none-eabi
+PLATFORM=launchxl
+
+DSLITE=$(UNIFLASH_CLI_BASE)/ccs_base/DebugServer/bin/DSLite
+PROG_BASE=0x47000
+
+include ../Makefile.common
+
+apps/%/build/$(TOCK_ARCH)/$(TOCK_ARCH).bin:
+	@make -C apps/$*
+
+flash: guard-UNIFLASH_CLI_BASE target/$(TARGET)/release/launchxl
+	@echo "Flashing target/$(TARGET)/release/launchxl"
+	@$(DSLITE) flash -c jtag/cc2652r1f.ccxml -e -f target/$(TARGET)/release/launchxl
+
+flash-%: apps/%/build/$(TOCK_ARCH)/$(TOCK_ARCH).bin guard-DSLITE
```

Comment:
I don't think we want this. In general, we try to keep a separation between kernel and apps. It's expected that if you want to flash the kernel, you `cd` into the proper board directory, and if you want to flash an app, `cd` into the proper userland/examples directory. If you want to keep it here to make testing things easier, that feels fine to me, but the proscribed method should be to cd into `userland/examples/blink` and `make flash TOCK_BOARD=launchxl`.

Now, having to specify TOCK_BOARD is a huge pain. So the overall solution should be to get Tockloader support for this platform, which will allow the default `flash` and `program` rules to apply: https://github.com/helena-project/tock/blob/master/userland/Program.mk

Reply:
It is mainly there to make testing easier. Adding support to tockloader would be awesome, will probably add it in the future. I'd prefer to keep it until proper tockloader support has been added :)